### ISSUE TITLE
Quick Booking buttons register touch but don't book.

### ISSUE THREAD

**Issue by MistaSee** [simplificator/sitincator#12]
Hi guys, looking to use Sitincator in our offices at Raspberry Pi but the final hurdle I'm having is that the quick booking buttons won't book the room. They register the press from the user, but nothing changes. 

Is there something about needing to be the owner of the calendar using the token, rather than my work account? The account it is currently using has access to add events to the calendar in question. 

Cheers, 

M.

**Comment by phylor** [comment]
Hi @MistaSee!

Does it display calendar events? Try to start it with `DISPLAY=":0" /home/pi/Sitincator/Sitincator --disable-pinch` and see if there are any errors.

You don't need to be the owner of the calendar. Having write access to the calendar is enough.

**Comment by jacobandersson86** [comment]
Hi! 
Just started to setup a Pi in order to use your app. I'm experiencing the same problem as mentioned above; pressing the "Quick Book" button won't work. I can see that the press itself is registered but nothing happends. I've tried starting with the command above from a SSH connection. It starts up the app, but no errors are printed...  I can se upcomming events and the full schedule, and if a book my meeting room in the google calendar, Sitincator displays it as busy. 

Do you have any ideas how to continue to troubleshoot this issue?? 

Many thanks in advance.

Jacob

**Comment by Zinggi** [comment]
Hey Jacob,

Are your rooms "resources" on google calendar? If so, you have the same issue that we are having currently.
We are aware of it, and I'm working on a branch that fixes it, but will require some more configuration.
The branch I'm working on is [here](https://github.com/simplificator/sitincator/tree/fix/booking), if you're interested.

However, we are currently very busy with other work that actually pays, so we can't allocate too much time for this project at the moment.

If you can help out, figuring out https://github.com/simplificator/sitincator/issues/22 would help me greatly.

Florian

**Comment by jacobandersson86** [comment]
Yes, that is correct that out meeting rooms are "resources". The meeting room resource has its own calendar, and I asked our G Suit Administrator to grant my user read/write permissions for this calendar. Then I repeated the "installation" process and it is now working!

I sure understand that you are busy. I guess my solution is rather a work-around than a proper fix but it works. I'm planning to add led-strips and a 3D printed casing for our display. I will contribute to this project by sharing .STL files for casings and code for LED strips etc. 

Many thanks again for creating Sitincator!!

**Comment by Zinggi** [comment]
I'm glad you got it working with that workaround, good that there is a solution for now.

In the branch I was working on you wouldn't need write permissions. But I had some issues building for the pie that I couldn't resolve yet, so you solution will do for now.
